ecf/scripts/stats/cam/jevs_cam_nam_firewxnest_grid2obs_stats_vhr_master.ecf

```diff
@@ -37,6 +37,7 @@ module load grib_util/${grib_util_ver}
 module load wgrib2/${wgrib2_ver}
 module load met/${met_ver}
 module load metplus/${metplus_ver}
+module load bufr/${bufr_ver}
```

Comment:
@PerryShafran-NOAA Quick question: Do we not need to add `module load bufr/${bufr_ver}` to the module load file for CAM stats because it's already been added to it for someone else's use?

Reply:
I believe that I saw that it's already there, but perhaps I should double check just to be sure.  I'm sure it is, because use of split_by_subset worked for me and I didn't add it myself.

Reply:
Yes, it's already there:

https://github.com/PerryShafran-NOAA/EVS/blob/feature/firewxnest_v2/dev/modulefiles/cam/cam_stats.sh

Reply:
Thanks, @PerryShafran-NOAA! 

CC @malloryprow @AndrewBenjamin-NOAA